packages/iris-grid/src/sidebar/conditional-formatting/ConditionalFormattingUtils.ts

```diff
@@ -0,0 +1,425 @@
+  config: FormatStyleConfig
+): string | undefined {
+  const { type, customConfig } = config;
+  switch (type) {
+    case FormatStyleType.NO_FORMATTING:
+      return undefined;
+    case FormatStyleType.POSITIVE:
+      return '#9fde6f';
+    case FormatStyleType.NEGATIVE:
+      return '#ff6087';
+    case FormatStyleType.WARN:
+      return '#f67f40';
+    case FormatStyleType.NEUTRAL:
+      return '#ffd95c';
+    case FormatStyleType.ACCENT_1:
+      return '#78dce8';
+    case FormatStyleType.ACCENT_2:
+      return '#ab9bf5';
+    case FormatStyleType.CUSTOM:
+      return customConfig?.background;
+    default:
+      return undefined;
+  }
+}
+
+export function getColorForStyleConfig(
+  config: FormatStyleConfig
+): string | undefined {
+  const { type, customConfig } = config;
+  switch (type) {
+    case FormatStyleType.NO_FORMATTING:
+      return undefined;
+    case FormatStyleType.POSITIVE:
+      return '#526a3f';
+    case FormatStyleType.NEGATIVE:
+      return '#802f44';
+    case FormatStyleType.WARN:
+      return '#663318';
+    case FormatStyleType.NEUTRAL:
+      return '#63562b';
+    case FormatStyleType.ACCENT_1:
+      return '#3f6469';
+    case FormatStyleType.ACCENT_2:
+      return '#554d72';
+    case FormatStyleType.CUSTOM:
+      return customConfig?.color;
+    default:
+      return undefined;
+  }
+}
+
+export function getStyleDBString(config: BaseFormatConfig): string | undefined {
+  const color = getColorForStyleConfig(config.style);
+  const bg = getBackgroundForStyleConfig(config.style);
+  if (color === undefined || bg === undefined) {
+    return undefined;
+  }
+  return `bgfg(\`${bg}\`, \`${color}\`)`;
+}
+
+function getNumberConditionText(config: BaseFormatConfig): string {
+  const { column, value, start, end } = config;
+  return getTextForNumberCondition(
+    column.name,
+    config.condition as NumberCondition,
+    value,
+    start,
+    end
+  );
+}
+
+function getStringConditionText(config: BaseFormatConfig): string {
+  const { column, value } = config;
+  return getTextForStringCondition(
+    column.name,
+    config.condition as StringCondition,
+    value
+  );
+}
+
+function getDateConditionText(config: BaseFormatConfig): string {
+  const { column, value } = config;
+  return getTextForDateCondition(
+    column.name,
+    config.condition as DateCondition,
+    value
+  );
+}
+
+export function getConditionDBString(config: BaseFormatConfig): string {
+  const { column } = config;
+
+  if (TableUtils.isNumberType(column.type)) {
+    return getNumberConditionText(config);
+  }
+  if (TableUtils.isTextType(column.type)) {
+    return getStringConditionText(config);
+  }
+
+  if (TableUtils.isDateType(column.type)) {
+    return getDateConditionText(config);
+  }
+
+  throw new Error('Invalid column type');
+}
+
+export function getLabelForNumberCondition(condition: NumberCondition): string {
+  switch (condition) {
+    case NumberCondition.IS_EQUAL:
+      return 'Is equal to';
```

Comment:
I think these should all be lower case (consistent with Advanced Filters):
```suggestion
      return 'is equal to';
```